### ISSUE TITLE
Create new Tag in bookmarks doesn't work

### ISSUE THREAD

**Issue by georgievgeorgi** [csete/gqrx#523]
Steps to reproduce the problem:
- Right click on Create new Tag in tags box
- Give it a name and confirm
- Click on some other tag
In this moment when the list gets refreshed the newly made tag disappears.

**Comment by csete** [comment]
Yes, I see this. It is still possible to create a new tag when adding a new bookmark.

**Comment by kolbma** [comment]
@georgievgeorgi 

I think this would be fixed in the changeset https://github.com/kolbma/gqrx/tree/bookmarks-ext
Although I've seen that I need one more change, that it doesn't edit the last tag in list any longer, but the newly added. Tomorrow ;-)

**Comment by darrenfreeman** [comment]
For a long time now, gqrx has segfaulted for me, after creating a new tag. I can only reliably create them by editing the bookmarks.csv manually.
